HoloPipelines/pipelines/pipelineAbdomenDicom2glb.py

```diff
@@ -0,0 +1,26 @@
+from components import compDicom2nifti
+from components import compHttpRequest
+from components import compNifti2numpy
+from components import compSeparateNumpy
+from components import compNumpyTransformation
+import sys
+
+
+def main(inputDicomPath, outputGlbFolderPath):
+    generatedNiftiPath = compDicom2nifti.main(inputDicomPath, "_temp.nii")
+    segmentedNiftiPath = compHttpRequest.sendFilePostRequest(
+        "http://localhost:5000/model",
```

Comment:
change this to read from a env variable

Reply:
this pipeline now takes url as an argument